pkg/purge/purge.go

```diff
@@ -0,0 +1,221 @@
+package purge
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/glasskube/glasskube/internal/clientutils"
+	"github.com/glasskube/glasskube/internal/config"
+	"github.com/glasskube/glasskube/internal/httperror"
+	"github.com/glasskube/glasskube/internal/releaseinfo"
+	"github.com/glasskube/glasskube/internal/telemetry"
+	"github.com/glasskube/glasskube/internal/telemetry/annotations"
+	"github.com/glasskube/glasskube/pkg/bootstrap"
+	"go.uber.org/multierr"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/schollz/progressbar/v3"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/restmapper"
+)
+
+type PurgeClient struct {
+	clientConfig *rest.Config
+	mapper       meta.RESTMapper
+	client       dynamic.Interface
+}
+
+type PurgeOptions struct {
+	Type                    bootstrap.BootstrapType
+	Url                     string
+	Latest                  bool
+	CreateDefaultRepository bool
+	DisableTelemetry        bool
+	Force                   bool
+}
```

Comment:
There currently isn't a way to change these options, so I think they should be removed for now.